src/mono/System.Private.CoreLib/src/ILLink/ILLink.Descriptors.xml

```diff
@@ -585,7 +585,10 @@
 
 		<!-- domain.c: mono_defaults.thread_class -->
 		<!-- FIXME: -->
-		<type fullname="System.Threading.Thread" preserve="fields" />
+		<type fullname="System.Threading.Thread" preserve="fields">
+			<!-- threads.c: call_thread_exiting -->
+			<method name="OnThreadExiting" />
```

Comment:
This would be better to do with `DynamicDependency(nameof(OnThreadExiting))]` on `Initialize` method

Reply:
Ah right, I keep forgetting we have that.